pkg/util/i18n/i18n.go

```diff
@@ -35,6 +35,7 @@ var knownTranslations = map[string][]string{
 		"default",
 		"en_US",
 		"fr_FR",
+		"jp_JP",
```

Comment:
The [ISO 639 language code of Japanese language](https://www.loc.gov/standards/iso639-2/php/code_list.php) is `ja`, so this should be changed to `ja_JP`.